app/views/idv/shared/_document_capture.html.erb

```diff
@@ -36,9 +36,7 @@
       in_person_outage_message_enabled: IdentityConfig.store.in_person_outage_message_enabled,
       in_person_outage_expected_update_date: IdentityConfig.store.in_person_outage_expected_update_date,
       us_states_territories: us_states_territories,
-      doc_auth_selfie_capture: {
-        enabled: IdentityConfig.store.doc_auth_selfie_capture_enabled,
-      },
+      doc_auth_selfie_capture: (Identity::Hostdata.env == 'prod' || !IdentityConfig.store.doc_auth_selfie_capture_enabled) ? false : doc_auth_selfie_capture,
```

Comment:
I think the controller already checks the hostdata env stuff in the concern? Or we wanna double check.

Reply:
yes, it does ... but added check on the way out to FE be extra careful